docusaurus/docs/iOS/05-ui-cookbook/15-long-press-to-focus.mdx

```diff
@@ -9,15 +9,45 @@ The StreamVideo SDK allows for interactive and intuitive user engagement during
 
 ## Overview
 
-- **Gesture Recognition**: Detect a long press on a participant's video feed.
 - **Focus Implementation**: Focus on the specified point on your video feed.
+- **Gesture Recognition**: Detect a long press or a tap on a participant's video feed.
 - **User Experience**: Intuitive interaction for a more immersive call experience.
 
 ## Implementing Long Press to Focus
 
+### Focusing on a point in the Video Feed
+
+In order to focus the camera on the desired we need to forward the request the WebRTC videoCapturer. Before we do that we need to create a call and then join it:
+```swift
+// Create the call with the callType and id
+let call = streamVideo.call(callType: "default", callId: "123")
+
+// Create the call on server side
+let creationResult = try await call.create()
+
+// Join the call
+let joinResult = try await call.join()
+```
+
+Once we are in the call, we can leverage the StreamVideo SDK to focus on a specific point on our **local video stream**:
+
+```swift
+// Retrieve the desired focus point
+let focusPoint: CGPoint = ...
```

Comment:
maybe better instead of "...", a compilable version (CGPoint with random values).